.circleci/config.yml

```diff
@@ -206,7 +229,7 @@ jobs:
   # Job for downloading the Code Climate test reporter
   cc-setup:
     docker:
-      - image: circleci/ruby:2.5
+      - image: circleci/ruby:2.6
```

Comment:
Actually, this should probably just be `circleci/ruby` so we don’t have to update it all the time.

Reply:
That makes sense! I updated this PR.